tests/unit/helpers/vendorHistoryExport.test.js

```diff
@@ -581,13 +586,12 @@ describe('exportCourseHistory', () => {
     groupSlotsByDate.onCall(2).returns([]);
     groupSlotsByDate.onCall(3).returns([]);
     groupSlotsByDate.onCall(4).returns([[courseSlotList[5]]]);
-    groupSlotsByDate.onCall(5).returns([[courseSlotList[6]]]);
+    groupSlotsByDate.onCall(5).returns([[singleCourseSlotList[0]]]);
```

Comment:
on est dans le cas not single course pourquoi on retour singleCourseSlotlist ?

Reply:
Bien vu, c'est une erreur. On appelle bien groupSlotsByDate 5 fois seulement